src/session/content/message_row/bubble.rs

```diff
@@ -284,15 +287,11 @@ impl MessageBubble {
             imp.sender_label.remove_css_class(&old_class);
         }
 
-        let color_class =
-            SENDER_COLOR_CLASSES[sender_id.map(|id| id as usize).unwrap_or_else(|| {
-                let mut s = DefaultHasher::new();
-                imp.sender_label.label().hash(&mut s);
-                s.finish() as usize
-            }) % SENDER_COLOR_CLASSES.len()];
-
-        imp.sender_label.add_css_class(color_class);
-        imp.sender_color_class.replace(Some(color_class.into()));
+        if let Some(sender_id) = sender_id {
+            let color_class = SENDER_COLOR_CLASSES[sender_id as usize % SENDER_COLOR_CLASSES.len()];
+            imp.sender_label.add_css_class(color_class);
+            imp.sender_color_class.replace(Some(color_class.into()));
+        }
```

Comment:
As I said in the group, this won't work because now the default sender color is not set anymore.

Anyway, I decided to make bigger changes to sender colors, which I'll work on another day. So, can you drop this commit for now?